index.cjs

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const modulePromise = import('./src/index.js')
+
+module.exports = {
```

Comment:
Could this be simplified? Something like:

```js
module.exports = import('./index.js')
```

And you just need to await the import in your project?